tpot/operators/selectors/select_from_model_r.py

```diff
@@ -46,22 +42,18 @@ class TPOTSelectFromModelR(Selector):
         'sklearn.ensemble':          ['ExtraTreesRegressor']
     }
     sklearn_class = SelectFromModel
-    arg_types = (float, int, float)
+    arg_types = (float, float)
     classification = False
 
     def __init__(self):
         pass
 
-    def preprocess_args(self, threshold, criterion, max_features):
+    def preprocess_args(self, threshold, max_features):
         threshold = min(1., max(0., threshold))
 
-        # Select criterion string from list of valid parameters
-        criterion_values = ['gini', 'entropy']
-        criterion_selection = criterion_values[criterion % len(criterion_values)]
-
         max_features = min(1., max(0., max_features))
 
         return {
-            'estimator': ExtraTreesRegressor(criterion=criterion_selection, max_features=max_features),
+            'estimator': ExtraTreesRegressor(max_features=max_features),
```

Comment:
Highlighting the second bugfix.

`ExtraTreesRegressor` only takes `mse` as a parameter fro criterion. I'd incorrectly set it up to use `'gini'` or `'entropy'` like `ExtraTreesClassifier` does.